src/test/java/apicurito/tests/steps/CommonSteps.java

```diff
@@ -62,6 +73,50 @@ public void importAPI(String pathtoFile) {
         ImportExportUtils.importAPI(new File(pathtoFile));
     }
 
+    @And("^generate and export fuse camel project$")
+    public void generateAndExportFuseCamelProject() {
+        File exportedFuseCamelProject = ImportExportUtils.exportFuseCamelProject();
+        assertThat(exportedFuseCamelProject)
+            .exists()
+            .isFile()
+            .has(new Condition<>(f -> f.length() > 0, "File size should be greater than 0"));
+    }
+
+    @And("^unzip and run generated fuse camel project$")
+    public void unzipAndRunGeneratedFuseCamelProject() throws Exception {
+        final Path sourceFolder = Paths.get("tmp" + File.separator + "download");
+        final File archive = sourceFolder.resolve("camel-project.zip").toFile();
+        final File destination = sourceFolder.resolve("camel-project").toFile();
+        ImportExportUtils.decompressZip(archive, destination);
+        log.info("Fuse Camel Project is decompressed to {}", destination);
+    }
+
+    @And("^check that project is generated correctly$")
+    public void checkThatProjectIsGeneratedCorrectly() throws VerificationException, MalformedURLException {
+        final Path sourceFolder = Paths.get("tmp" + File.separator + "download");
+        final Path destination = sourceFolder.resolve("camel-project");
+
+        // Create a new thread and run the generated camel project with maven commands
+        final MavenUtils mavenUtils = MavenUtils.forProject(destination).forkJvm();
+        final ExecutorService executorService = Executors.newSingleThreadExecutor();
+        executorService.submit(() -> {
+            try {
+                mavenUtils.executeGoals("clean", "package");
+                mavenUtils.executeGoals("spring-boot:run");
+            } catch (VerificationException e) {
+                log.error("Error during running the app");
+                log.error("Error message: ", e);
+            }
+        });
+
+        // Open the web browser to verify the returned code is 200 OK and the generated camel project is correct
+        final String testUrl = "http://localhost:8080/openapi.json";
+        assertThat(Https.doesUrlReturnOK(testUrl).waitFor()).isTrue();
+        assertThat(HttpUtils.readFileFromURL(new URL(testUrl))).contains("A brand new API with no content.  Go nuts!");
```

Comment:
I would check for something else since that description is by default in openapi. Check for something different.